app/helpers/sessions_helper.rb

```diff
@@ -0,0 +1,42 @@
+module SessionsHelper
+
+  def log_in(user)
+    session[:user_id] = user.id
+  end
+
+  def current_user
+    if session[:user_id]
+      @current_user ||= User.find_by(id: session[:user_id])
+    end
+  end
+
+  def logged_in?
+    !current_user.nil?
```

Comment:
`current_user.present?` why do you always use `!something_something.nil?` when you have `something_something.present?` I told you this in the other PR a few times

Reply:
modified wherever i saw it in previous PR. somehow I always skip one. will be more careful with those. ^_^
solved. ✓